src/Chromamboo/Providers/Notification/AtlassianCiSuiteBuildStatusNotificationProvider.cs

```diff
@@ -73,21 +71,19 @@ public void Register(string planKey)
             this.presentationService.Update(buildsDetails);
         }
 
-
-        private BuildDetail GetBuildDetails(JToken jToken)
+        private BuildDetail GetBuildDetails(JToken jsonToken)
         {
-            var key = jToken["key"].Value<string>();
+            var key = jsonToken["key"].Value<string>();
```

Comment:
You know I'm lazy. It was much faster to edit the variable's name than editing our stylecop rules.